### ISSUE TITLE
Fill in ancestry/migration events section

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/msprime#1272]
Currently blank section [here](https://tskit-dev.github.io/msprime-docs/main/ancestry.html#migration-events)

The "Population structure" section in the tutorial chapter might serve as a useful guide: https://tskit-dev.github.io/msprime-docs/main/ancestry.html#migration-events

**Comment by jeromekelleher** [comment]
@awohns, any chance you could pick this up soon?

**Comment by awohns** [comment]
Yep will do

**Comment by awohns** [comment]
@jeromekelleher, As a simple example of migrations in `msprime`, I've made a stepping stone model as discussed:
```
N = 10
demography = msprime.Demography.stepping_stone_model(
    [20] * N,
    migration_rate=0.1,
    boundaries=True)
ts = msprime.sim_ancestry({0: 1, N - 1: 1},
                          demography=demography, record_migrations=True, record_full_arg=True, random_seed=6)
```
Without recording the full arg, the tree looks like this:
![Screenshot 2021-03-29 at 12 34 16](https://user-images.githubusercontent.com/10664424/112869375-15786e00-908b-11eb-8c87-0f4d8ed65eb6.png)

I then plotted the population migration paths of samples 0 and 2 back in time to their MRCA, giving colours to each of the 10 populations from the viridis continuous colormap.
![image](https://user-images.githubusercontent.com/10664424/112868982-a8fd6f00-908a-11eb-9e55-a0268688166c.png)

I've also plotted the population colours on the tree (with some helpful pointers from @hyanwong!): 
![image](https://user-images.githubusercontent.com/10664424/112870066-ced74380-908b-11eb-973e-dcdfb159a539.png)

I think this is the right sort of thing to show how migrations work, but there's a bit of plotting code. Is it worth including that code in the section?
